specification/apicenter/ApiCenter.Management/main.tsp

```diff
@@ -37,7 +39,7 @@ enum Versions {
 }
 
 @@projectedName(Apis.createOrUpdate::parameters.resource, "json", "payload");
-@@projectedName(Apis.createOrUpdate::parameters.resource, "client", "payload");
+@@clientName(Apis.createOrUpdate::parameters.resource, "payload");
```

Comment:
@timotheeguerrin if you look above this line, these are the projected name jsons I'm still seeing